core/src/components/content/content.tsx

```diff
@@ -329,7 +329,8 @@ export class Content implements ComponentInterface {
             'inner-scroll': true,
             'scroll-x': scrollX,
             'scroll-y': scrollY,
-            'overscroll': (scrollX || scrollY) && forceOverscroll
+            'overscroll': (scrollX || scrollY) && forceOverscroll,
+            'scroll-snap': isPlatform('ios')
```

Comment:
Can we change this so it only activates when the collapsible large title is in use?